AlexWang-16/alexwang-16.github.io
# Fix event heading not right aligned

Fixes #25 
# Purpose of update
Fix events heading not right aligned

# Change log
* Add pull-right css class to events section
* Add "clear-fix" css class to address issues with pull-right css class breaking layout

# Test Plan
1. go to alexwang.ca
2. On navbar, click on "events"
3. observe that events heading is right aligned

Test this on mobile to ensure the heading

Expected result:
The user should see events heading right aligned with proper spacing between divider line, the heading, and its contents.

# Additional notes
